packages/ag-charts-community/src/chart/chartContext.ts

```diff
@@ -80,9 +80,13 @@ export class ChartContext implements ModuleContext {
         this.syncManager = syncManager;
         this.zoomManager = chart.zoomManager;
         this.domManager = new DOMManager(container);
-        scene?.setContainer(this.domManager);
```

Comment:
The code here is slightly convoluted due to the 're-use resources from a destroyed chart instance' case, but I don't think we should be dealing with the raw elements here. The code here shouldn't care about specific elements are involved in the composition of the chart.

Reply:
> The code here shouldn't care about specific elements are involved in the composition of the chart.
Neither should the `Scene` object - it only needs an `HTMLCanvasElement`. I think that this is a good opportunity to this tidy up.

The initialisation of the chart-context shouldn't be so complex. The  `'re-use resources from a destroyed chart instance'` case isn't any different to a create-or-update logic.

Reply:
Done. Change reverted.